pikaur/config.py

```diff
@@ -1,10 +1,16 @@
 import os
 
+from .core import running_as_root
+
 VERSION = '0.6-dev'
 
-CACHE_ROOT = os.path.expanduser('~/.cache/pikaur/')
-AUR_REPOS_CACHE = os.path.join(CACHE_ROOT, 'aur_repos')
-BUILD_CACHE = os.path.join(CACHE_ROOT, 'build')
+if running_as_root():
+    CACHE_ROOT = '/var/lib/pikaur'
```

Comment:
why it accidentially turned from /var/cache/ to /var/lib/ ?

Reply:
Oh, woops, I was doing some experiments with StateDirectory= and forgot to change it. Fixed.